### DISCUSSION TITLE
socket not working (Error : websocket.js:124 WebSocket connection to 'wss://outline.example.com/realtime/?EIO=3&transport=websocket' failed: )

### DISCUSSION THREAD

**Discussion by himanshuparmar3355** [outline/outline#3433] [Self Hosting]
I installed docker setup, and we have Apache server for reverse proxy.

I already tried this
https://github.com/outline/outline/discussions/3097#discussion-3871398

In browser showing following error.
websocket.js:124 WebSocket connection to 'wss://outline.example.com/realtime/?EIO=3&transport=websocket' failed:

**Comment by almereyda** [comment]
Hi @himanshuparmar3355,

exciting to witness your experiment to run this also behind an unsupported Reverse Proxy, such as Apache `httpd`.

As your posting does not contain a question, allow me to infer it from your statement:

> Were you asking for working examples of similar setups with Outline in Docker behind an Apache Webserver that have working WebSockets connections?

And to be perfectly clear about your usage of the snippets that you linked to:

> Did you replace the `**YOUR.DOMAIN**`, `**YOUR LOCAL IP ADDRESS**` and `YOUR LOCAL IP ADDRESS` stanzas with your values?

These could be `example.com` and one of the container names that the internal Docker DNS resolves to for your `outline` container, or its IP. Another question I'm asking myself is:

> Is the Webserver running as a container attached to the same network as the Outline container, like the default bridge or a custom one, to reach it directly, or is the Outline container mapped to (local)host ports, which in turn are accessed by the web server, which is scheduled outside of Docker on the host system (or someplace else)?

Many more things could be deviating from the specification here, why it's good to be explicit about your assumptions. It will be very helpful, if you can contribute more details from your webserver and Outline configuration.

Additionally, I would suggest you also contact the author of the linked discussion, and ask them for more ideas, since we are out of the supported scope of the application here.
